src/main/java/com/flowpowered/nbt/regionfile/SimpleRegionFileReader.java

```diff
@@ -77,7 +78,7 @@
                 raf.readFully(data);
                 ByteArrayInputStream in = new ByteArrayInputStream(data);
                 InflaterInputStream iis = new InflaterInputStream(in);
```

Comment:
See. SimpleRegionFileReader does not force NBTInputStream to implement a compression method that only it (region files) ever use. :)